spec/features/login_spec.rb

```diff
@@ -0,0 +1,29 @@
+feature "Login" do
+  scenario "User can login" do
+    visit "/login"
+    User.create(username: "test_user", email: "test@test.com", password: "test")
+    fill_in "email", with: "test@test.com"
+    fill_in "password", with: "test"
+    click_button "Enter"
+    expect(page).to have_content "Welcome"
```

Comment:
Feel maybe this should welcome the user by name - otherwise could potentially have logged in the wrong user and the test wouldn't know?